### ISSUE TITLE
stellar transactions no longer supported

### ISSUE THREAD

**Issue by daviddelikat** [keybase/client#25982]
when I open my wallet I get the message that stellar is not supported and I should 'export' my balance.  I don't see anything about this anywhere else is there a problem with my account?  how do I export my balance?
<img width="580" alt="Screen Shot 2023-10-25 at 11 01 46 AM" src="https://github.com/keybase/client/assets/91501/27d3a72d-6d4f-45c4-8f67-a3991f44db80">

**Comment by mathieujobin** [comment]
I'm not a Stellar fan, but I am sad to see features being removed.
Keybase had the best integration for sending coins.

**Comment by jmsaunders** [comment]
> You will need to migrate any Stellar Lumens (XLM) you have by merging to a different wallet. I would highly recommend [using LOBSTR to merge your XLM](https://merge.lobstr.co/) so it's seamless and you don't lose any. This will solve the problems you are experiencing. There are numerous tutorials on this that can help you if you're unsure.

Do you have any links for tutorials handy?

**Comment by Beasta** [comment]
Zoom bought keybase and they'll slowly pwn it into a dumpster fire

**Comment by JoNilsson** [comment]
Very sad to see robust, highly functional features being removed from Keybase.   No official notice either.

**Comment by mathieujobin** [comment]
yeah, I am not specifically a fan of Stellar, or a Crypto advocate.
but Keybase had succeed bringing a user-friendly crypto wallet, a thing that no other companies have done, and actually Meta/Facebook was promising of doing for a while.

I think more currencies should have been added.

please revert the removal.

**Comment by mathieujobin** [comment]
@jmsaunders @loudersoft The fund does not need to be migrated. the wallet is still valid and you can still access your private key within Keybase. only the UI features of sending/receiving money has been removed. so you don't need to move your funds. just need your favorite client and use your private key to send, your public key (or wallet address) to receive.

that said, I did move mine to my Ledger hardware wallet.

**Comment by sbehnken** [comment]
Having the same issue. Keybase gives this message. But nothing else. I have my paper key handy but there's literally no option to move things. Can anyone help with this?
<img width="664" alt="Screen Shot 2024-05-23 at 2 19 49 PM" src="https://github.com/keybase/client/assets/42148216/d9fa49b2-4857-4ffa-a446-ad61b0d5f371">

**Comment by mathieujobin** [comment]
@sbehnken use the command line tool
`keybase wallet balances` to show your current balances
`keybase wallet export` to get your Private keys

**Comment by nsilve** [comment]
@mathieujobin having access to this info, how could we export the balances to an external wallet (e.g. Coinbase)?

**Comment by mathieujobin** [comment]
@nsilve 

https://developers.stellar.org/docs/build/guides/basics/send-and-receive-payments

**Comment by mathieujobin** [comment]
Someone could also for the client and try reverting the removal of the Stellar wallet

**Comment by nsilve** [comment]
> https://developers.stellar.org/docs/build/guides/basics/send-and-receive-payments

That was really helpful, it worked!
Thank you very much!
